deployments/puppet/Rakefile

```diff
@@ -0,0 +1,13 @@
+require 'rspec'
+require 'rspec-puppet/rake_task'
+require 'puppetlabs_spec_helper/rake_tasks'
+
+begin
+  if Gem::Specification::find_by_name('puppet-lint')
+    require 'puppet-lint/tasks/puppet-lint'
+    PuppetLint.configuration.ignore_paths = ["spec/**/*.pp", "vendor/**/*.pp"]
+    task :default => [:rspec, :lint]
```

Comment:
lint is a separate make target and ci job, so is this necessary?

Reply:
I'm not too familiar with ruby/rake so not sure... this was just copied from the smart agent. But running `rake spec` doesn't seem to be failing on the same lint errors that `puppet-lint` would. Thoughts?